pkg/test/framework/components/istio/operator.go

```diff
@@ -384,13 +398,6 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
 	}
 
-	// For multicluster, configure direct access so each control plane can get endpoints from all API servers.
-	if ctx.Clusters().IsMulticluster() {
-		if err := i.configureDirectAPIServerAccess(ctx, cfg); err != nil {
-			return nil, err
-		}
-	}
-
```

Comment:
Removing this will break [Multi-Primary](https://istio.io/latest/docs/setup/install/multicluster/multi-primary/)

Reply:
@frankbu multi-primary also share secrets? I did not see that from docs . if they do, what secrets get shared?

Reply:
https://istio.io/latest/docs/setup/install/multicluster/multi-primary/#enable-endpoint-discovery